src/core/lib/iomgr/work_serializer.cc

```diff
@@ -33,6 +35,20 @@ struct CallbackWrapper {
   const DebugLocation location;
 };
 
+// First 32 bits indicate ownership of the WorkSerializer, next 32 bits are
+// queue size (i.e., refs).
+static uint64_t MakeRefPair(uint32_t owners, uint32_t size) {
```

Comment:
These can be private methods of `WorkSerializerImpl`.

Reply:
This is already within an anonymous namespace. Any reason to prefer private methods?